### DISCUSSION TITLE
New paper accepted for publication

### DISCUSSION THREAD

**Discussion by robinthibaut** [robinthibaut/skbel#5] [Announcements]
A new [paper](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2022WR033045) has been accepted for publication in Water Resources Research. This research paper uses the SKBEL package to its full extent.

Abstract:
Temperature logs are an important tool in the geothermal industry. Temperature measurements from boreholes are used for exploration, system design, and monitoring. The number of observations, however, is not always sufficient to fully determine the temperature field or explore the entire parameter space of interest. Drilling in the best locations is still difficult and expensive. It is therefore critical to optimize the number and location of boreholes. Due to its higher spatial resolution and lower cost, four-dimensional (4D) temperature field monitoring via time-lapse Electrical Resistivity Tomography (ERT) has been investigated as a potential alternative. We use Bayesian Evidential Learning (BEL), a Monte Carlo-based training approach, to optimize the design of a 4D temperature field monitoring experiment. We demonstrate how BEL can take into account various data source combinations (temperature logs combined with geophysical data) in the Bayesian optimal experimental design (BOED). To determine the optimal data source combination, we use the Root Mean Squared Error (RMSE) of the predicted target in the low dimensional latent space where BEL is solving the prediction problem. The parameter estimates are accurate enough to use in BOED. Furthermore, the method is not limited to monitoring temperature fields and can be applied to other similar experimental design problems. The method is computationally efficient and requires little training data. For the considered optimal design problem, a training set of only 200 samples and a test set of 50 samples is sufficient.

Citation:
Thibaut, Robin, Nicolas Compaire, Nolwenn Lesparre, Maximilian Ramgraber, Eric Laloy, and Thomas Hermans (Oct. 2022). “Comparing Well and Geophysical Data for Temperature Monitoring within a Bayesian Experi-mental Design Framework”. In: Water Resources Research. issn: 0043-1397. doi:10.1029/2022WR033045. url: https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2022WR033045
